Helpers/RLMRealm+Notifications.m

```diff
@@ -65,4 +65,26 @@ - (void)deleteObjectsWithNotification:(id<NSFastEnumeration>)array
     [self deleteObjects:array];
 }
 
+#pragma mark - private helpers
+
+- (BOOL)rbq_containsObject:(RLMObject *)object {
+    if (!object) {
+        return NO;
+    }
+    
+    Class objectClass = [object class];
```

Comment:
The class name needs to use the same logic that is in `RLMObject+Utilities.m` because I want to maintain support for Swift.

The class name for an `RLMObject` in Swift is different: `AppName.ClassName`